modules/openapi-generator/src/main/resources/typescript-fetch/runtimeSagasAndRecords.mustache

```diff
@@ -1,6 +1,8 @@
 /* tslint:disable */
 /* eslint-disable */
 
+// Some imports not used depending on template conditions
+// @ts-ignore
 import {fromJS as originalFromJS, isIndexed, List, Map as ImmMap, RecordOf} from 'immutable';
```

Comment:
**suggestion:** It looks like the only unused import here is `ImmMap` - and it is unconditionally unused, literally never mentioned in the template expect for the import. So perhaps we can just remove it instead of ts-ignore'ing?

Reply:
Ok.